examples/rpc/rpc_from_drg_module/main.tf

```diff
@@ -44,41 +41,31 @@ module "vcn_local" {
       description       = "Terraformed - User added Routing Rule to remote vcn through DRG"
     },
   ]
+
   providers = {
     oci = oci
   }
   
 }
 
-module "subnet_local" {
-  source          = "github.com/oracle-terraform-modules/terraform-oci-tdf-subnet"
-  
-  default_compartment_id  = var.compartment_id
-  vcn_id                  = module.vcn_local.vcn_id
-  vcn_cidr                = var.local_vcn_cidr
-  
-  subnets = {
-    local_rpc_subnet               = {
-      compartment_id    = null
-      defined_tags      = null
-      freeform_tags     = null
-      dynamic_cidr      = false
-      cidr              = var.local_vcn_cidr
-      cidr_len          = null
-      cidr_num          = null
-      enable_dns        = true
-      dns_label         = "localrpcsubnet"
-      private           = true
-      ad                = null
-      dhcp_options_id   = null
-      route_table_id    = module.vcn_local.nat_route_id
-      security_list_ids = null
-    },
-  }
+resource "oci_core_subnet" "subnet_local" {
+  #Required
+  cidr_block     = var.local_vcn_cidr
+  compartment_id = var.compartment_id
+  vcn_id         = module.vcn_local.vcn_id
+
+  #Optional
+  display_name               = "sub-local"
+  dns_label                  = "rpcsublocal"
+  prohibit_public_ip_on_vnic = true
+  route_table_id = module.vcn_local.nat_route_id
+  freeform_tags    = var.freeform_tags
 }
 
+
 module "drg_local" {
-  source = "../../../modules/drg"
+  source = "oracle-terraform-modules/vcn/oci//modules/drg"
```

Comment:
There's a double // here

Reply:
That's not a typo. It is the correct syntax to access a submodule directly from the registry.

Reply:
Ok.